executor/point_get.go

```diff
@@ -302,6 +302,9 @@ func (e *PointGetExecutor) get(ctx context.Context, key kv.Key) ([]byte, error)
 func encodeIndexKey(e *baseExecutor, tblInfo *model.TableInfo, idxInfo *model.IndexInfo, idxVals []types.Datum, tID int64) (_ []byte, err error) {
 	sc := e.ctx.GetSessionVars().StmtCtx
 	for i := range idxVals {
+		if idxVals[i].IsNull() {
```

Comment:
If we skip the null value, we built a non exists index key, I think it is not the right way to fix the bug.

Reply:
Why does the index key not exist? This loop is used to cast the `idxVals` to target types, which is unnecessary for null type values.
After casting, all null values will also be encoded into the key.

Reply:
But if the index has a null value, it is not encoded as unique index, the rowid is appended to the key.

Reply:
I mean we should let the caller know the index value has a null value, we can not point get this key.

Reply:
okay

Reply:
Updated. PTAL @coocood